test/functional/ios/file-movement-specs.js

```diff
@@ -102,9 +102,12 @@ describe('file movements - pullFile and pushFile @skip-real-device', function ()
           });
         };
         if (parseFloat(sdk) >= 8) {
-          getSimUdid('6', sdk, env.CAPS, function (err, udid) {
+          xcode.getVersion(function(err, xcodeVersion){
```

Comment:
sounds like this will get superseded by 1.5 soon anyway, but: our style guide is spaces around function definition parentheses. `function (err, xcodeVersion) {`